templates/ingress.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "cmak.name" . | quote }}
+  labels:
+    {{- include "cmak.labels" . | nindent 4 }}
+{{- with .Values.ingress.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.ingress.tls }}
+  tls:
+  {{- range .Values.ingress.tls }}
```

Comment:
In values.yaml `tls` field is boolean, but this template uses it as an array / list. 
But then again, why do we have a single host but multiple secrets ? I believe ingress should allow to configure a single host and a single secret.